### ISSUE TITLE
[FR] Notification for stopwatch

### ISSUE THREAD

**Issue by woj-tek** [SimpleMobileTools/Simple-Clock#85]
Google clock application has neat feature - showing elapsed time in stopwatch mode in notification. This allows peaking how much time elapsed without opening the application.

**Comment by WPFilmmaker** [comment]
I agree, very useful feature! Alarm Klock (opensource on F-Droid) which is unfortunately no longer maintaned and has vulnerablities, do that and it is very useful!
